bulk_merger.rb

```diff
@@ -2,9 +2,9 @@
 
 class BulkMerger
   def self.approve_unreviewed_pull_requests!(list: nil)
-    puts "Searching for Dependabot PRs for gem '#{gem_name}'"
+    puts "Searching for PRs for gem '#{query_string}'"
```

Comment:
This would read oddly in your use case, wouldn't it?

`Searching for PRs for gem "Relax authorized"`

I'd suggest swapping `for gem` for `containing`.

Reply:
Oh yeah, good spot! Fixed 👍. I've also removed a sentence that didn't seem to have any practical impact on using the repo.